openedx2zim/xblocks_extractor/problem.py

```diff
@@ -72,11 +80,11 @@ def download(self, c):
             )
             if "success" in answers_content:
                 """
-                    #IMPROUVEMENT connection , same as hint ?
+                    #IMPROUVEMENT instance_connection , same as hint ?
```

Comment:
fix the typo in comment maybe?

Reply:
Fixed the typo. But kept the comment as that'll allow us to look into #27 later on